modules/30-variables/20-magic-numbers/description.en.yml

```diff
@@ -0,0 +1,62 @@
+---
+
+name: Magic numbers
+theory: |
+
+  Recall one of the previous lessons:
+
+  ```javascript
+  let dollarsCount = 50 * 1.25; // 62.5
+  let rublesCount = dollarsCount * 60; // 3750
+
+  console.log(rublesCount);
+  ```
+
+  From the developer's point of view, such code "smells". It means that the code is hard to comprehend. Even now, when you look at the numbers `60` and `1.25`, you are likely to ask: "What are those numbers?" And imagine reading it in a month! And how will a new programmer who hasn't seen the code before understand it? In our example, you reconstruct the context through proper naming, but in real life, the code is much more complicated and it is often impossible to guess the meaning of numbers.
```

Comment:
in real life, the code is much more complicated and it is often impossible => in real life, the code is much more complicated**, and it is often impossible**